scripts/cli/tests/2_key_management.js

```diff
@@ -0,0 +1,194 @@
+// Set options as a parameter, environment variable, or rc file.
+require = require("esm")(module /*, options*/);
+module.exports = require("../util/init.js");
+
+const {
+  path,
+  fs,
+  ApiPromise,
+  WsProvider,
+  createIdentities,
+  initMain,
+  colors,
+  cliProg,
+  updateStorageSize,
+  blockTillPoolEmpty,
+  distributePoly
+} = require("../util/init.js");
+
+let {
+  STORAGE_DIR,
+  nonces,
+  synced_block,
+  synced_block_ts,
+  current_storage_size,
+  transfer_amount,
+  n_accounts,
+  master_keys,
+  signing_keys,
+  claim_keys,
+  fail_type,
+  fail_count,
+  block_sizes,
+  block_times,
+  entities,
+  initial_storage_size
+} = require("../util/init.js");
+
+async function main() {
+  // Schema path
+  const filePath = path.join(__dirname + "/../../../polymesh_schema.json");
```

Comment:
The following 5 or 6 lines are duplicated in each file. What do you think if we put them into a common place?

Reply:
alright would work on that

Reply:
I created a object called reqImports which holds all the functions that would be usually imported so only the one object needs to be imported from all files now.